packages/azure-kusto-data/src/index.ts

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Moving index might also be a breaking change. Build and check

Reply:
We are doing a breaking change - what will this break?

Reply:
I take it back, if the file is updated correctly then nothing.